### PR TITLE
Nexmo

### PR DESCRIPTION
**Pull request by AlessioCasco**
developer --> added nexmo

### CODE REVIEW

**Comment by mxxcon** [comment]
How does this site fit in https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria ?

**Comment by AlessioCasco** [comment]
Nexmo is one of the first operators for sms and voice over API, it deals with data, you use the 2fa to log in into your dashboard where you can find all the messages and calls you sent and received

**Comment by Carlgo11** [comment]
No doc link?

**Comment by AlessioCasco** [comment]
I had a look around but I don't see any doc or info specifically related to their dashboard & 2fa.
I can load a screenshot of the log-in page if you need a prove.

**Comment by Carlgo11** [comment]
Yes a screenshot would be great.
(Blur out ant identifying information that might be visible in the screenshot)

**Comment by AlessioCasco** [comment]
Screenshot attached

2fa_nexmo

File: _data/developer.yml [hunk 1]
```diff
@@ -200,6 +200,12 @@ websites:
       img: newrelic.png
       tfa: No
 
+    - name: Nexmo
+      url: https://www.nexmo.com/
+      img: nexmo.png
+      tfa: Yes
+      sms: Yes
```
Comment (jamcat22) [inline _data/developer.yml:207]: @AlessioCasco From the screenshot, it looks like a phone call is an available method of 2FA. Can you please add the line `phone: Yes` if this is the case?

**Comment by AlessioCasco** [comment]
@jamcat22 You're right, the phone call is a method they do offer.
I updated the profile, 

Thanks

**Pull request merged**